grammars/hrm.cson

```diff
@@ -32,29 +32,25 @@
         'name': 'keyword.control.c'
   }
   {
-    'match': '^    (COMMENT) +([0-9]+)'
+    'match': '^ *(COMMENT) +([0-9]+)'
     'captures':
       '1':
         'name': 'entity.name.function.preprocessor.c'
       '2':
-        'name': 'string.quoted.double.include.c'
+        'name': 'constant.numeric.c'
```

Comment:
That's true, but I kind of want to visually differentiate between comments (which refer to things in the code) and labels (which don't)…  Is there another syntax type you know of that would be appropriate but still differently-coloured?